### ISSUE TITLE
getScriptArgs API behavior differs from standard Ghidra in headless mode

### ISSUE THREAD

**Issue by qkaiser** [mandiant/Ghidrathon#68]
### Description

If you use Ghidrathon in headless mode with both a `-preScript` and `-postScript`, the call to `getScriptArgs()` does not work as expected.

If you launch an analysis with the following command line:

```sh
analyzeHeadless /tmp/out out -import /bin/ls -scriptPath -preScript script.py arg_a arg_b -postScript script.py arg_x arg_y
```

The API call `getScriptArgs()` will return arguments `arg_a` and `arg_b` for both the `preScript` and `postScript`.

Of course it gets worse if the user does not use `preScript` since the argument won't be visible to `postScript` at all.

### Expected behavior

The API call `getScriptArgs()` should return `["arg_a", "arg_b"]` for `preScript` and `["arg_x", "arg_y"]` for `postScript`.

### Environment Info

- OS: `Linux (Debian 11)`
- Python version: `3.10.12`
- Ghidra version: `10.3`
- Ghidrathon version: `2.2.1`
- openjdk version `17.0.7" 2023-04-18`

### Reproduction Steps

```sh
rm -rf /tmp/out && mkdir /tmp/out
echo "print([x for x in getScriptArgs()])" > /tmp/script.py
analyzeHeadless /tmp/out out -import /bin/ls -scriptPath -preScript script.py arg_a arg_b -postScript script.py arg_x arg_y
```

<details>
<summary>Command line output</summary>
<pre>
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment (build 17.0.7+7-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 17.0.7+7-Debian-1deb11u1, mixed mode)
INFO  Using log config file: jar:file:/opt/ghidra/Ghidra/Framework/Generic/lib/Generic.jar!/generic.log4j.xml (LoggingInitialization)  
INFO  Using log file: /app/.ghidra/.ghidra_10.3_PUBLIC/application.log (LoggingInitialization)  
INFO  Loading user preferences: /app/.ghidra/.ghidra_10.3_PUBLIC/preferences (Preferences)  
INFO  Searching for classes... (ClassSearcher)  
INFO  Class search complete (869 ms) (ClassSearcher)  
INFO  Initializing SSL Context (SSLContextInitializer)  
INFO  Initializing Random Number Generator... (SecureRandomFactory)  
INFO  Random Number Generator initialization complete: NativePRNGNonBlocking (SecureRandomFactory)  
INFO  Trust manager disabled, cacerts have not been set (ApplicationTrustManagerFactory)  
INFO  HEADLESS Script Paths:
    /opt/ghidra/Ghidra/Features/SystemEmulation/ghidra_scripts
    /opt/ghidra/Ghidra/Features/GnuDemangler/ghidra_scripts
    /opt/ghidra/Ghidra/Processors/DATA/ghidra_scripts
    /opt/ghidra/Ghidra/Processors/JVM/ghidra_scripts
    /opt/ghidra/Ghidra/Features/MicrosoftCodeAnalyzer/ghidra_scripts
    /opt/ghidra/Ghidra/Features/Python/ghidra_scripts
    /opt/ghidra/Ghidra/Features/VersionTracking/ghidra_scripts
    /opt/ghidra/Ghidra/Features/Base/ghidra_scripts
    /opt/ghidra/Ghidra/Processors/Atmel/ghidra_scripts
    /opt/ghidra/Ghidra/Features/Decompiler/ghidra_scripts
    /opt/ghidra/Ghidra/Features/FunctionID/ghidra_scripts
    /opt/ghidra/Ghidra/Features/PDB/ghidra_scripts
    /opt/ghidra/Ghidra/Processors/8051/ghidra_scripts
    /opt/ghidra/Ghidra/Processors/PIC/ghidra_scripts
    /tmp
    /opt/ghidra/Ghidra/Features/FileFormats/ghidra_scripts
    /opt/ghidra/Ghidra/Extensions/ghidrathon/ghidra_scripts
    /opt/ghidra/Ghidra/Features/BytePatterns/ghidra_scripts (HeadlessAnalyzer)  
INFO  HEADLESS: execution starts (HeadlessAnalyzer)  
INFO  Creating project: /tmp/out/out (HeadlessAnalyzer)  
INFO  Creating project: /tmp/out/out (DefaultProject)  
INFO  REPORT: Processing input files:  (HeadlessAnalyzer)  
INFO       project: /tmp/out/out (HeadlessAnalyzer)  
INFO  IMPORTING: /bin/ls (HeadlessAnalyzer)  
INFO  Using Loader: Executable and Linking Format (ELF) (AutoImporter)  
INFO  Using Language/Compiler: x86:LE:64:default:gcc (AutoImporter)  
INFO  IMPORTING: Loaded 0 additional files (HeadlessAnalyzer)  
INFO  SCRIPT: /tmp/script.py (HeadlessAnalyzer)  
['arg_a', 'arg_b']
INFO  ANALYZING all memory and code: /bin/ls (HeadlessAnalyzer)  
INFO  DWARF external debug information found: ExternalDebugInfo [filename=61a544c35b9dc1d172d1a1c09043e487326966.debug, crc=d165fd8a, hash=6461a544c35b9dc1d172d1a1c09043e487326966] (ExternalDebugFilesService)  
INFO  Unable to find DWARF information, skipping DWARF analysis (DWARFAnalyzer)  
INFO  hit non-returning function, restarting decompiler switch analyzer later (DecompilerSwitchAnalyzer)  
INFO  Packed database cache: /tmp/analysis-Ghidra/packed-db-cache (PackedDatabaseCache)  
INFO  Applied data type archive: generic_clib_64 (ApplyDataArchiveAnalyzer)  
INFO  -----------------------------------------------------
    ASCII Strings                              0.421 secs
    Apply Data Archives                        0.227 secs
    Call Convention ID                         0.012 secs
    Call-Fixup Installer                       0.015 secs
    Create Address Tables                      0.055 secs
    Create Address Tables - One Time           0.021 secs
    Create Function                            0.042 secs
    DWARF                                      0.008 secs
    Data Reference                             0.053 secs
    Decompiler Switch Analysis                 0.089 secs
    Decompiler Switch Analysis - One Time      2.851 secs
    Demangler GNU                              0.021 secs
    Disassemble Entry Points                   1.020 secs
    ELF Scalar Operand References              0.326 secs
    Embedded Media                             0.019 secs
    External Entry References                  0.000 secs
    Function Start Search                      0.043 secs
    Function Start Search After Code           0.012 secs
    Function Start Search After Data           0.009 secs
    GCC Exception Handlers                     0.412 secs
    Non-Returning Functions - Discovered       0.240 secs
    Non-Returning Functions - Known            0.005 secs
    Reference                                  0.159 secs
    Shared Return Calls                        0.112 secs
    Stack                                      1.741 secs
    Subroutine References                      0.046 secs
    Subroutine References - One Time           0.000 secs
    x86 Constant Reference Analyzer            1.973 secs
-----------------------------------------------------
     Total Time   9 secs
-----------------------------------------------------
 (AutoAnalysisManager)  
INFO  REPORT: Analysis succeeded for file: /bin/ls (HeadlessAnalyzer)  
INFO  SCRIPT: /tmp/script.py (HeadlessAnalyzer)  
['arg_a', 'arg_b']
INFO  ANALYZING changes made by post scripts: /bin/ls (HeadlessAnalyzer)  
INFO  REPORT: Post-analysis succeeded for file: /bin/ls (HeadlessAnalyzer)  
INFO  REPORT: Save succeeded for: /ls (out:/ls) (HeadlessAnalyzer)  
INFO  REPORT: Import succeeded (HeadlessAnalyzer) 
</pre>
</details>

**Comment by mike-hunhoff** [comment]
Thank you for reporting @qkaiser . I haven't had a chance to triage this locally but it may be related #65 . I'll hopefully have a fix for #65 coming next week and then I can circle back here to see if the issue is resolved.

**Comment by mike-hunhoff** [comment]
@qkaiser this bug is fixed in our latest release: https://github.com/mandiant/Ghidrathon/releases/tag/v3.0.0.
